decidim-budgets/app/packs/stylesheets/budgets.scss

```
@@ -4,12 +4,8 @@
   }
 
   &-summary {
-    @apply bg-tertiary;
-
-    box-shadow: 0 4px 6px 0 rgba(211, 211, 211, 0.25);
-
     &__content {
-      @apply p-4 md:p-0;
+      @apply bg-tertiary p-4 h-[78px] md:h-full md:p-0;
```

This is the change that we need to do after (after removing the javascript file and the line with the new file in  decidim_budget.js). 

```suggestion
      @apply bg-tertiary p-4 md:h-full md:p-0;
```

Let me know if that works

---

It works, thanks.